xblockmufi/public/edit.js

```diff
@@ -32,37 +41,39 @@ function XblockMufiEdit(runtime, element) {
             },
             error: function buttonSaveOnError() {
                 runtime.notify('error', {});
-            }
+            },
         });
+        /* eslint-enable camelcase */
         return false;
     });
 
     // TEXT EDITOR JS
-    $('li.specialButton').click(function(e) {
+    /* eslint-disable no-invalid-this*/
+    $('li.specialButton').click(function (e) {
```

Comment:
can you try making this `mousedown`?

Reply:
Why?

I'd like to minimize the amount of user-facing changes contained in this;
I'd like it to mostly be "just working w/ the new system", as much as possible.

Why would we want buttons to trigger on `mousedown` instead of `click` (and release) though?

Reply:
mousedown here too, this is used by instructors to layout the transcription for comparison with the learners.

Reply:
Done.